src/app/(private)/bridge/patients-and-ecmos/page.tsx

```
@@ -8,18 +8,130 @@ import { PatientDataTable } from "../patients/data-table";
 import { Button } from "~/components/ui/button";
 import Link from "next/link";
 import { ArrowUpRight } from "lucide-react";
+import Map from "./map"
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)